lightstep-tracer-common

```diff
@@ -1 +1 @@
-Subproject commit a2ac69205156c196ef5dbe7037fda7091580d1e9
+Subproject commit b1bdd09872f4e7d6439f613181997dedb1783458
```

Comment:
I checked-out master (in ls-tracer-common) and pulled. That gives me a different SHA so go figure.